### ISSUE TITLE
Users report: Improve flow, add percentages

### ISSUE THREAD

**Issue by mikeymckay** [mikeymckay/coconut#180]

**Comment by gmcressman** [comment]
The title of this report is strange to me. It's actually not a report about users, but is a report about response times. Shouldn't the title reflect that?

**Comment by mikeymckay** [comment]
Yes. We've actually renamed all the reports. I can't remember what this one
is called now and I'm not at my laptop.

**Comment by mikeymckay** [comment]
![image](https://cloud.githubusercontent.com/assets/12015/12582926/103f8a4c-c451-11e5-9603-a07e93427f77.png)
